charts/spire/charts/spire-server/templates/server-resource.yaml

```diff
@@ -79,9 +79,17 @@ metadata:
     app.kubernetes.io/component: server
 spec:
   {{- if not .Values.autoscaling.enabled }}
+  {{- if eq .Values.dataStore.plugin "connect" }}
+  {{- if and (not (include "has-any-upstreamauthority" .)) (gt (int .Values.replicaCount) 1) }}
```

Comment:
I guess that the disk-based authority would suffer from the same issue (possibly not if you're not using ReadWriteMany shared storag), since they would have separate roots.

Reply:
I answered that in the comment above - it is configured to read from a shared secret.

Reply:
Thanks - makes sense if it's a secret.